packages/embarkjs/src/blockchain.js

```diff
@@ -187,13 +187,16 @@ Blockchain.doConnect = function(connectionList, opts, doneCb) {
         return doneCb(err);
       }
       const currentProv = self.blockchainConnector.getCurrentProvider();
-      if (opts.warnAboutMetamask && currentProv && currentProv.isMetaMask) {
+      if (opts.warnAboutMetamask && (
+          (currentProv && currentProv.isMetaMask) || 
+          (window.ethereum && window.ethereum.isMetaMask)
+        )) {
         // if we are using metamask, ask embark to turn on dev_funds
         // embark will only do this if geth is our client and we are in
         // dev mode
         if (opts.blockchainClient === 'geth') {
           // TODO find a way to share the port number
-          console.warn("%cNote: There is a known issue with Geth that may cause transactions to get stuck when using Metamask. Please log in to the cockpit (http://localhost:55555/embark?enableRegularTxs=true) to enable a workaround. Once logged in, the workaround will automatically be enabled.", "font-size: 2em");
+          console.warn("%cNote: There is a known issue with Geth that may cause transactions to get stuck when using Metamask. To enable a workaround, start an Embark console (run command `embark console` in your terminal) or open the dashboard in Cockpit (http://localhost:55555), then type in the console command `devtxs on`. To disable the workaround, type `devtxs off` in the console.", "font-size: 2em");
```

Comment:
I've experienced the same "stuck Txs" problem when I'm *not* using Metamask, so maybe the text and logic should reflect it's being a problem with geth more generally?

Reply:
Updated so that metamask is not a requirement to show that warning.